build-system/tasks/helpers.js

```diff
@@ -118,6 +122,19 @@ async function compileCoreRuntime(options) {
   await doBuildJs(jsBundles, 'amp.js', options);
 }
 
+/**
+ * Compiles the "core" utilies used by all bento extensions
+ *
+ * Outputs 2 scripts:
+ * 1) for direct consumption in the browser
+ * 2) for consumption by npm package users
+ * @param {!Object} options
```

Comment:
type nit: `!` no longer has meaning because we use TypeScript for typechecking instead of CC
```suggestion
 * @param {Object} options
```

Reply:
updated